src/python/grpcio_health_checking/MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 include grpc_version.py
 include health_commands.py
-graft grpc
```

Comment:
I don't understand how the change on this line relates to the rest of the pull request. Have I missed something in the commit messages?